README.md

```diff
@@ -41,9 +67,6 @@ This repository is licensed under GNU-AGPLv3 since v1.5.0, find our license [her
 **Where does the data come from?**
```

Comment:
Shouldn't we add a link to the official FAQ as well?

Reply:
Yes, good point! :)